sdk/formrecognizer/azure-ai-formrecognizer/MIGRATION_GUIDE.md

```diff
@@ -0,0 +1,639 @@
+                print("......Selection mark is '{}' and has a confidence of {}".format(
+                    selection_mark.state,
+                    selection_mark.confidence
+                ))
+
+    print("-----------------------------------")
+```
+
+Analyze custom document with `3.2.x`:
+```python
+with open(path_to_sample_documents, "rb") as f:
+    poller = document_analysis_client.begin_analyze_document(
+        model=model_id, document=f
+    )
+result = poller.result()
+
+for idx, document in enumerate(result.documents):
+    print("--------Analyzing document #{}--------".format(idx + 1))
+    print("Document has type {}".format(document.doc_type))
+    print("Document has document type confidence {}".format(document.confidence))
+    print("Document was analyzed with model with ID {}".format(result.model_id))
+    for name, field in document.fields.items():
+        field_value = field.value if field.value else field.content
+        print("......found field of type '{}' with value '{}' and with confidence {}".format(field.value_type, field_value, field.confidence))
+
+
+# iterate over tables, lines, and selection marks on each page
+for page in result.pages:
+    print("\nLines found on page {}".format(page.page_number))
+    for line in page.lines:
+        print("...Line '{}'".format(line.content))
+    for word in page.words:
+        print(
+            "...Word '{}' has a confidence of {}".format(
+                word.content, word.confidence
+            )
+        )
+    if page.selection_marks:
+        print("\nSelection marks found on page {}".format(page.page_number))
+        for selection_mark in page.selection_marks:
+            print(
+                "...Selection mark is '{}' and has a confidence of {}".format(
+                    selection_mark.state, selection_mark.confidence
+                )
+            )
+
+for i, table in enumerate(result.tables):
+    print("\nTable {} can be found on page:".format(i + 1))
+    for region in table.bounding_regions:
+        print("...{}".format(i + 1, region.page_number))
+    for cell in table.cells:
+        print(
+            "...Cell[{}][{}] has text '{}'".format(
+                cell.row_index, cell.column_index, cell.content
+            )
+        )
+print("-----------------------------------")
+```
+
+### Training a custom model
+
+Differences between the versions:
+- Files for building a new model for version `3.2.x` can be created using the labeling tool found [here][fr_labeling_tool].
+- In version `3.1.x` the `use_training_labels` keyword argument was used to indicate whether to use labeled data when creating the custom model.
+- In version `3.2.x` the `use_training_labels` keyword argument is not supported since training must be carried out with labeled training documents. In order to extract key-value pairs from a document, please refer to the prebuilt model "prebuilt-document" which extracts entities, key-value pairs, and layout from a document. 
```

Comment:
sorry to bring this up again, but I don't think we should mention key-value pairs as this was not what it was referred to in v3.1.x. maybe something like?

"Additionally train without labels is now replaced with the prebuilt model "prebuilt-document"...."